vpd-tool/include/utils.hpp

```diff
@@ -0,0 +1,124 @@
+#pragma once
+
+#include "tool_types.hpp"
+
+#include <nlohmann/json.hpp>
+#include <sdbusplus/bus.hpp>
+#include <sdbusplus/exception.hpp>
+
+#include <iostream>
+
+namespace vpd
+{
+namespace utils
+{
+/**
+ * @brief An API to read property from Dbus.
+ *
+ * The caller of the API needs to validate the validatity and correctness of the
```

Comment:
API description can be more descriptive. Some typos in this text.

Reply:
done

Reply:
validate the validity and correctness - all the 3 words points to same meaning.

Reply:
this API is duplicated from dbus_utility.
the statement looks fine for me.